recipes/android-ndk/all/conanfile.py

```diff
@@ -60,7 +60,7 @@ def source(self):
         pass
 
     def build(self):
-        if self.version in ['r23', 'r23b', 'r23c', 'r24', 'r25']:
+        if self.version not in ['r21d', 'r21e', 'r22', 'r22b']:
```

Comment:
```suggestion
        major, _ = self._ndk_major_minor
        if major < 23:
```

it's probably more future proof, given that there is currently a PR aiming to add few old versions.

Reply:
Good point, was not aware of those additional PRs. Nice to have the integration here on GitHub that checks them against each other. Will commit your suggestion as-is, thank you!